core/arch/arm/kernel/entry_a64.S

```diff
@@ -173,6 +185,14 @@ FUNC _start , :
 	msr	vbar_el1, x0
 	isb
 
+#ifdef CFG_PAN
+	read_feat_pan x0
```

Comment:
I suppose this could go into the `init_pan` macro too.

Reply:
moved into `init_pan`